docs/cli/cli.mdx

```diff
@@ -25,10 +24,16 @@ Note: You can also specify the token with the `--token=<your-repository-token>`
 
 ## Upload Command
 
-Use the `upload` command to upload screenshots stored in your <Tag>./screenshots</Tag> directory.
+Use the `upload` command to upload screenshots stored in your <code>./screenshots</code> directory.
 
 <RunPkgCommand command="argos upload ./screenshots" />
 
+### Debug option
+
+You can enable debug mode by setting the `DEBUG` environment variable.
```

Comment:
Actually it's not the good place. It's `DEBUG=@argos-ci/core` and it works for all our SDKs.